apps/studio/src/assets/styles/app/query-editor.scss

```diff
@@ -112,6 +102,24 @@
 
   }
 
+  .CodeMirror {
+    padding-bottom: 0.25rem;
+  }
+
+  .CodeMirror-linenumbers {
+    width: 22px;
```

Comment:
Can these be `rem` based, or are we trying to match Tabulator's widths?

Reply:
Yes, it can. I was trying to match the tabulator, but it should be easy to convert that.